Integration doc (#5)
brains-and-beards/mimic-docs: advanced/integrations.md

# Integrations

> Mimic app is good solution for mobile developers which want to write automatic and integration tests for application. They can do this without any testing database and testing api.

Reviewer: ```suggestion
Mimic app is good solution for mobile developers who would like to write automatic and integration tests for their applications. They can do this without any testing database and testing API.
```

> In Brains and Breads we are using Detox library to to write integration test in react native apps. [More info](https://github.com/wix/Detox)

Reviewer: ```suggestion
In Brains&Breads we are using Detox library to write integration tests for React Native apps. [More info](https://github.com/wix/Detox)
```

Reviewer 2: How about linking the [Detox library] directly rather than adding "More info" link at the end of the sentence?

> ## Integration React Native App with Detox

Reviewer: ```suggestion
## Integrating React Native Apps with Detox
```

> Detox is an end-to-end testing and automation library for mobile apps. It tests your mobile app while it's running in a real device/simulator, interacting with it just like a real user. We'll show you how you can use it with Mimic to test your API with mocked endpoints.

Reviewer: ```suggestion
Detox is an end-to-end testing and automation library for mobile apps. It tests your mobile app while it's running on a real device/simulator, interacting with it just like a real user. We'll show you how you can use it with Mimic to test your API with mocked endpoints.
```
